cpp/src/utilities/cython.cu

```diff
@@ -696,6 +696,61 @@ std::unique_ptr<cy_multi_edgelists_t> call_egonet(raft::handle_t const& handle,
   }
 }
 
+// Wrapper for random_walks() through a graph container
+// to expose the API to cython.
+//
+template <typename vertex_t, typename edge_t>
+std::enable_if_t<cugraph::experimental::is_vertex_edge_combo<vertex_t, edge_t>::value,
+                 std::unique_ptr<random_walk_ret_t>>
```

Comment:
This looks nice to me, shouldn't we apply this to other algorithms as well?

Reply:
We need to evaluate the ultimate plan for this first, and that should be a separate PR, I believe.